.github/workflows/changelog-verify.yml

```diff
@@ -46,7 +46,11 @@ jobs:
         # Set outputs using the command.
         run: |
           bash ./.githooks/pre-receive "${{needs.changedfiles.outputs.all}}" 2>&1 | tee output.txt
-          echo "all=$(cat output.txt | jq -sR .)" >> $GITHUB_OUTPUT
+          VALIDATION_OUTPUT=$(cat output.txt)
+          echo "all=$(echo "$VALIDATION_OUTPUT" | jq -sR .)" >> $GITHUB_OUTPUT
+          if [[ "$VALIDATION_OUTPUT" == *"failed"* ]]; then 
+            exit 1
+          fi
```

Comment:
I think this presenting here because you merged main in using a fast-forward (i.e. no merge commit):

![image](https://github.com/pokt-network/pocket/assets/600733/446b97a0-fdef-4cfd-9a96-0ec7c5b17216)

I'm making a note to add something about this to the code review guidelines. I'm not 100% sure that's the correct or perhaps only place for it though. It makes sense in terms of giving feedback to the author if they didn't know, but I think the git workflow should be outlined in developer (or perhaps contributing (?)) documentation. In this case I think the advise is to use `git merge --no-ff ...` when merging main into PR branches.

Reply:
(The only other scenario I can imagine at the moment that would cause this would be if you cherry-picked those commits but the commit messages don't show any evidence of that (e.g. "cherry-picked from ..."; did you destroy it? 🧐:stuck_out_tongue_closed_eyes:).)

Reply:
FWIW, the consequence here has been that to finish this review, I had to go commit by commit starting from where I last reviewed.